services/server/test/server.spec.ts

```diff
@@ -2283,83 +2322,69 @@ describe("Server", function () {
       }
     });
 
-    const verifierAllianceTestLibrariesManuallyLinked = require("./verifier-alliance/libraries_manually_linked.json");
     it(verifierAllianceTestLibrariesManuallyLinked._comment, async () => {
       await verifierAllianceTest(
         server,
-        chai,
         storageService,
         localSigner,
         defaultContractChain,
         verifierAllianceTestLibrariesManuallyLinked
       );
     });
 
```

Comment:
Not necessarily in this PR but the part below here can be nicely refactored. A lot of repetitive code here. Just import everyting and loop over it.

Reply:
Yes. You can always do `--grep` in mocha to run a particular test or skip manually. It's ugly and really cumbersome IMO to write the long variable names and the whole test function for each test.

Reply:
In my opinion, it is fine to have some repetitive code in tests (at least to some degree). The reason is that it makes the tests self-contained. And self-contained tests are more understandable and maintainable. I think creating tests in loops contradicts this principle.

In this specific case, I would rather write the content of `_comment` directly into the `it` call to achieve this. The parameters of the `verifierAllianceTest` should be reduceable with a proper server configuration setup for the tests, because they are always the same (only the testcase parameter differs).

Anyway, I would rather leave refactoring this part (regardless which way we go) in another PR.

Reply:
Closing this and I'm fine either way, not a strong opinion.